### DISCUSSION TITLE
SX1262 vs SX126x

### DISCUSSION THREAD

**Discussion by mrakowRC** [jgromes/RadioLib#956] [General]
I'm pretty new at this microcontroller stuff so please forgive me. I am currently using the atmega4804 and I have everything transmitting correctly when I use SX1262. The only problem that I'm having now is that the quiet sound doesn't work at all. The quiet noise does work when I reprogram my device so I'm curious if the pins are getting reset somehow in my wakeup function.

When I change to SX126x my quiet sound comes back and works just fine.

If anyone could help me understand the difference between SX1262 and SX126x that would be nice

**Comment by jgromes** [comment]
> The only problem that I'm having now is that the quiet sound doesn't work at all. The quiet noise does work

I am extremely confused right now - what is "quiet sound"/"quiet noise"? How is it related to this library?

> If anyone could help me understand the difference between SX1262 and SX126x that would be nice

SX126x is a base class for SX1262 or SX1268 and is in no way intended to be used by the end user (as its documentation states).
